protocols/relay/src/priv_client/handler.rs

```diff
@@ -128,6 +129,17 @@ pub struct Handler {
         >,
     >,
 
+    wait_for_reserve_outbound_stream: VecDeque<mpsc::Sender<transport::ToListenerMsg>>,
+    reserve_futs: futures_bounded::WorkerFutures<
+        (),
+        Result<outbound_hop::Output, outbound_hop::UpgradeError>,
+    >,
```

Comment:
Definition of `WorkerFutures` looks right.

Reply:
I don't have a strong opinion on whether `substream_timeout` and `max_concurrent_streams` should be configurable. You @thomaseizinger. For the sake of keeping the API surface small, I am fine settling on constants for now. In case we ever want to configure them, we can do so in a non-breaking way.

Reply:
I'll change to constants usage